edgeai-apps-utils/suite/bookworm/debian/rules

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/make -f
+
+export SOC=am62x
```

Comment:
Makes sense. 
I talked to the edgeai team. They have agreed to do it, but they said that it will require some work. They said that it may not be possible in the near-future, but they'll try to squeeze it in 9.2 release.